src/cache.service.ts

```diff
@@ -18,23 +19,36 @@ export const MESSAGES = {
 
 export type CacheValueFactory<T> = () => Promise<T>;
 
-// @dynamic
+/**
+ * @description Check if it's an HttpResponse
+ * @param {any} data - Variable to test
+ * @return {boolean} - data from cache
+ */
+function isHttpResponse(data: any): boolean {
```

Comment:
I prefer `const isHttpResponse = (data) => {...}`

Reply:
Fixed in next commit